docs/perf_analyzer.md

```diff
@@ -400,3 +400,39 @@ can achieve by using shared memory in your application. Use
 By default perf_analyzer uses HTTP to communicate with Triton. The GRPC
 protocol can be specificed with the -i option. If GRPC is selected the
 --streaming option can also be specified for GRPC streaming.
+
+## Using C API directly
+
+### Prerequisite
```

Comment:
`Pull the Triton SDK and the Inference Server container images on target machine`.